src/test/java/org/cactoos/proc/BiProcOfTest.java

```diff
@@ -130,4 +132,25 @@ void worksWithBiProc() throws Exception {
             )
         ).affirm();
     }
+
+    @Test
+    void worksWithTwoProcs() throws Exception {
+        final AtomicInteger fst = new AtomicInteger();
+        final AtomicInteger snd = new AtomicInteger();
+        new Assertion<>(
+            "Must execute BiProc with two Procs",
+            new BiProcOf<>(
+                new BiProcOf<>(
```

Comment:
@andreoss why `BiProcOf` wrapped into a `BiProcOf`? Just one is enough, no?

Reply:
@victornoel Done